deploy-service/common/src/main/java/com/pinterest/deployservice/scm/GithubManager.java

```diff
@@ -55,7 +55,8 @@ public class GithubManager extends BaseManager {
 
     public Map<String, String> headers = new HashMap<String, String>();
 
-    public GithubManager(String token, String appId, String appPrivateKeyKnox, String appOrganization, String apiPrefix, String urlPrefix) throws Exception {
```

Comment:
I did the same thing for PhabricatorManager so that the typeName from configs gets populated when getTypeName() is called in ConfigHelper.java on line 144